AppDashboard/dashboard.py

```diff
@@ -105,16 +105,21 @@ def render_template(self, template_file, values=None):
     if values is None:
       values = {}
 
-    owned_apps = self.dstore.get_owned_apps()
-    self.helper.update_cookie_app_list(owned_apps, self.request, self.response)
+    is_cloud_admin = self.helper.is_user_cloud_admin()
+    if is_cloud_admin:
+      apps_user_is_admin_on = self.dstore.get_application_info()
+    else:
+      apps_user_is_admin_on = self.helper.get_owned_apps()
+    self.helper.update_cookie_app_list(apps_user_is_admin_on, self.request, self.response)
```

Comment:
Ah! I thought I had anticipated all of them that would need a list instead, I'll add .keys() and it'll be good.